frontend/packages/dev-console/integration-tests/support/pages/topology/topology-actions-page.ts

```diff
@@ -0,0 +1,76 @@
+import { nodeActions } from '../../constants/topology';
+import { modal } from '../../../../../integration-tests-cypress/views/modal';
+
+export const topologyActions = {
+  selectAction: (action: nodeActions | string) => {
+    switch (action) {
+      case nodeActions.EditApplicatoinGrouping || 'Edit Application Grouping': {
```

Comment:
what you had before was fine then having `||` here ,  my doubt was why we need condition as both looks similar to me 'Edit Application Grouping' and nodeActions.EditApplicatoinGrouping shouldn't the const as well have same values?

Reply:
Sometimes we use to pass them from gherkin scripts like
user selects "Move Sink" from context menu
and some times without "" (double quotes) so it should work for both. So I used these technique

Reply:
ah does that matter in switch case string is string right check here https://jsfiddle.net/invincibleJai/3tov1ags/3/